code-of-conduct.md

```diff
@@ -1,2 +1,55 @@
-Jupyter accessibility workshops are events within the Project Jupyter community.
-They follow [Project Jupyter's Code of Conduct](https://jupyter.org/conduct/).
+# Jupyter Accessibility Workshops Code of Conduct
+
+Jupyter accessibility workshops are events within the Project Jupyter community. They follow [Project Jupyter's Code of Conduct](https://jupyter.org/conduct/).
+
+Project Jupyter's Code of Conduct mentions that events may have additional specifications to ensure that people are kept safe in real-time interactions. For Jupyter accessibility workshops, we would like to specify the following:
```

Comment:
```suggestion
As per Project Jupyter's Code of Conduct - events may have additional specifications to ensure that people are kept safe in real-time interactions. For Jupyter accessibility workshops, we the following addendums to the Code of Conduct have been made and will be enforced
```

Reply:
This adds more strength to the additions and highlight that there are mechanisms to enforce them